daphne/src/lib.rs

```diff
@@ -225,6 +236,45 @@ pub struct ProblemDetails {
     pub detail: Option<String>,
 }
 
+/// Global DAP parameters common across tasks.
+//
+// TODO spec: `max_window_size`, `min_duration_start`, and `max_duration_end`
+//            should be reflected to the DAP spec.
+#[derive(Deserialize, Serialize)]
+#[serde(try_from = "ShadowDapGlobConfig")]
+pub struct DapGlobConfig {
+    /// Maximum window size permitted in CollectReq.
+    /// Prevents Collectors from requesting wide range or reports.
+    pub max_window_size: u64,
```

Comment:
I think a better name would be "max_batch_duration", as it aligns better with "min_batch_duration".

Reply:
I've changed the naming of this and the other variables in this struct.